config/routes.rb

```diff
@@ -11,6 +11,10 @@
     get "/cookies", to: "cookies#show"
     get "/accessibility-statement", to: "accessibility_statement#show"
 
+    # Redirect for deleted question and page (301 is default)
+    get "/urgent-medical-help", to: redirect("/need_help_with")
+    get "/get-help-from-nhs", to: redirect("/need_help_with")
```

Comment:
```suggestion
    get "/get-help-from-nhs", to: redirect("/need-help-with")
```
Looks like there's a small typo in the redirect URL (I typed in wrong in my last comment 🤦 ).

Reply:
That'll teach me to copy-and-paste without checking! Thanks Bruce.

Reply:
There's also the same typo on the line above this one

Reply:
🤦 x 10